### PR TITLE
CONTRIBUTING.rst: add note about idling on IRC

### PR DESCRIPTION
**Pull request by mbolivar**
Make sure it's clear that IRC is not an instant-response medium.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -185,6 +185,13 @@ finding, or that have similar ideas for changes or additions.  Send a message
 to the `Zephyr-devel mailing list`_ to introduce and discuss your idea with
 the development community.
 
+Please note that it's common practice on IRC to be away from the
+channel, but still have a client logged in to receive traffic. If you
+ask a question from a particular person and they don't answer, **try
```
Comment (dbkinder) [inline CONTRIBUTING.rst]: change "from" to "to"

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -185,6 +185,13 @@ finding, or that have similar ideas for changes or additions.  Send a message
...
+to stay signed in to the channel** if you can, so they have time to
+respond to you. This is especially important given the many different
+timezones Zephyr developers live in.
```
Comment (dbkinder) [inline CONTRIBUTING.rst]: Maybe add something like:

    If you don't get a timely response on IRC, try sending a
    message to the mailing list instead.

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Review by rgundi — APPROVED** [review]
Address David's comments and i think it should be fine

**Comment by mbolivar** [comment]
Both fixed, thanks!

**Review by dbkinder — APPROVED** [review]
thanks

**Pull request merged**